### ISSUE TITLE
In multi-module project, file paths are not relative to module's base

### ISSUE THREAD

**Issue by tyrel** [SonarOpenCommunity/sonar-cxx#768]
I have a multi-module sonar project. So in the project's sonar-project.properties, I've got this:
```properties
sonar.modules=module,module2,module3
```
 In each module, I'm specifying this in sonar-project.properties:
```properties
sonar.cxx.cppcheck.reportPath=cppcheck.xml
```

The cppcheck.xml was generated by cppcheck running in that module's base directory (the same directory sonar-project.properties and cppcheck.xml are in).

cppcheck.xml contains paths relative to this base directory, not surprisingly:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<results version="2">
  <cppcheck version="1.61"/>
  <errors>
  <error id="nullPointer" severity="error" msg="Possible null pointer dereference: vectorTable" verbose="Possible null pointer dereference: vectorTable">
    <location file="src/exception.cpp" line="72"/>
  </error>
...
```
When I run sonar-runner, CxxCppCheckSensor looks for these files, e.g. `src/exception.cpp`, relative to the project base directory, not relative to the module's base directory:
```
INFO: Sensor CxxCppCheckSensor
DEBUG: Using pattern 'cppcheck.xml' to find reports
INFO: Processing report '/path/to/project/module/cppcheck.xml'
INFO: Parsing 'Cppcheck V2' format
INFO: Cannot find the file '/path/to/project/src/exception.cpp', skipping violations
INFO: Added report '/path/to/project/module/cppcheck.xml' (parsed by: CppcheckParserV2)
INFO: CppCheck Errors processed = 0
INFO: Sensor CxxCppCheckSensor (done) | time=369ms
```
You can see there it's looking for `/path/to/project/src/exception.cpp` instead of `/path/to/project/module/src/exception.cpp`

**Comment by jmecosta** [comment]
this should be address in the last snapshot, can you give a try:
https://ci.appveyor.com/project/SonarOpenCommunity/sonar-cxx/branch/master/artifacts

further more, now its possible for you to use absolute paths in the reports

**Comment by tyrel** [comment]
Confirmed, this is fixed in the 0.9.5 snapshot.
